app/Http/Requests/ProfileFormRequest.php

```diff
@@ -30,6 +30,7 @@ public function rules()
             'phone' => 'nullable|string|max:11',
             'address' => 'nullable|string|max:200',
             'position' => 'nullable|string|max:10',
+            'role' => 'required|in:0, 1',
```

Comment:
is this not required when normal user login, right?
only when current login user is admin